### PR TITLE
Add string enums for enums while generating TypeScript.

### PR DESCRIPTION
**Pull request by rmjohnson**
Closes https://github.com/FacilityApi/FacilityJavaScript/issues/19

### CODE REVIEW

**Comment by rmjohnson** [comment]
CC: @ejball @ddunkin

File: src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs [hunk 1]
```diff
@@ -109,6 +109,23 @@ public override CodeGenOutput GenerateOutput(ServiceInfo service)
...
+						typeNames.Add(enumInfo.Name);
+						code.WriteLine();
+						WriteJsDoc(code, enumInfo);
+						using (code.Block($"export enum {enumInfo.Name} {{", "}"))
+						{
+							foreach (var value in enumInfo.Values)
+							{
+								code.WriteLineSkipOnce();
+								WriteJsDoc(code, value);
+								code.WriteLine($@"{value.Name} = ""{value.Name}"",");
```
Comment (ejball) [inline src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs:124]: Should this use single quotes?

File: VersionHistory.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Version History
 
+### 2.2
```
Comment (ejball) [inline VersionHistory.md:3]:
```suggestion
### 2.2.0
```

`Directory.Build.props` will need to be updated as well.

File: src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs (hunk 1 above)
Comment (ddunkin) [inline src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs:124]: Nit: we use `'` in the other generated code.

**Review by ddunkin — COMMENTED** [review]
👍

File: src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs (hunk 1 above)
Comment (rmjohnson) [reply src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs]: Updated: https://github.com/FacilityApi/FacilityJavaScript/pull/26/commits/28f54864e1d092063a70eccf0e6a5e35da85ca14

Comment (rmjohnson) [reply src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs]: Updated: https://github.com/FacilityApi/FacilityJavaScript/pull/26/commits/28f54864e1d092063a70eccf0e6a5e35da85ca14

File: VersionHistory.md (hunk 2 above)
Comment (rmjohnson) [reply VersionHistory.md]: Updated: https://github.com/FacilityApi/FacilityJavaScript/pull/26/commits/70887627d4db7840ad65fbbd03ee7b9e4f464c80

**Comment by rmjohnson** [comment]
@ejball, @ddunkin ready for further review =)

**Review by ddunkin — APPROVED** [review]

**Review by ejball — APPROVED** [review]

**Comment by rmjohnson** [comment]
@ejball, @ddunkin, I don't actually have permission to merge, would one of you mind hitting merge? 😅

**Comment by ejball** [comment]
@rmjohnson Permission granted!

**Pull request merged**